src/pages/FundraisingClaim/FundraisingClaim.tsx

```diff
@@ -57,28 +49,38 @@ export const FundraisingClaimPage = () => {
     fetchClaimIsActive()
   })
 
-  const handleClaim = useCallback(() => {
-    const claim = async () => {
-      try {
-        const txReceipt = await sherClaim.claim()
-        console.log(txReceipt)
-        await refreshClaimableAmount()
-      } catch (error) {
-        console.log(error)
-      }
+  const handleClaim = useCallback(async () => {
+    try {
+      const txReceipt = await sherClaim.claim()
+      console.log(txReceipt)
+      refetchFundraisePosition()
+    } catch (error) {
+      console.log(error)
     }
+  }, [sherClaim, refetchFundraisePosition])
+
+  if (!fundraisePositionData?.fundraisePosition) return null
+
+  const sherAmount =
+    fundraisePositionData?.fundraisePosition?.reward && BigNumber.from(fundraisePositionData.fundraisePosition.reward)
+  const formattedSherAmount = ethers.utils.commify(ethers.utils.formatUnits(sherAmount))
 
-    claim()
-  }, [sherClaim, refreshClaimableAmount])
+  const claimableAt = new Date(Number(fundraisePositionData.fundraisePosition.claimableAt))
```

Comment:
Might need to update here to transform `claimableAt` from seconds to milliseconds, in case the indexer will store the timestamps the same way they are coming from the contract. That remains to be seen when the indexer is up.

Reply:
For sure. I left this here more as a placeholder. We might need to come up with a couple formatting utility functions to use across the app.